src/components/footerMenu/FooterMenu.tsx

```diff
@@ -0,0 +1,22 @@
+import { NavLink } from 'react-router-dom'
+import './footerMenu.css'
+
+export function FooterMenu({ isAuthenticated }:{ isAuthenticated:boolean }) {
```

Comment:
It is better to create an interface holding definitions of component props, later when you have more than one prop it will be more readable